### PR TITLE
starts_with & ends_with rules

### PR DESCRIPTION
**Pull request by DeveloperMarius**
Hey,

I created starts_with & ends_with rules. It works with stings, arrays and numbers.

Maybe this would be interesting to be a general rule.

~ Marius

### CODE REVIEW

File: src/Rules/EndsWith.php [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+            }else{
+                $last_value = $value[sizeof($value)-1];
+            }
+            return $last_value === $compare_with;
+        }
+
+        return false;
+    }
+
+    /**
```
Comment (dave-redfern) [inline src/Rules/EndsWith.php]: This docblock is redundant and can be removed.

File: src/Rules/StartsWith.php [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+            }else{
+                $first_value = $value[0];
+            }
+            return $first_value === $compare_with;
+        }
+
+        return false;
+    }
+
+    /**
```
Comment (dave-redfern) [inline src/Rules/StartsWith.php]: This docblock is redundant and can be removed.

**Review by dave-redfern — CHANGES_REQUESTED** [review]
Thank you for your contribution. Could you please re-format your code to follow PSR-12, being sure to include braces around any if/else statements etc, and include `declare(strict_types=1)` in the opening `<?php` tags of all files.

**Comment by DeveloperMarius** [comment]
Hey @dave-redfern,
thank you for your feedback. I did the changes you asked for.

~ Marius

**Pull request merged**